src/utils/get-hover-color/index.ts

```diff
@@ -0,0 +1,31 @@
+import { hcl, rgb } from 'd3-color';
+
+interface ColorModifiers {
+  brighter: number;
+  darker: number;
+  opacity: number;
+}
+
+const getHoverColor = (color: string, colorModifies: ColorModifiers) => {
```

Comment:
colorModifies -> colorModifie**rs** 
Probably auto-complete kept suggesting you this